cocos/spine/skeleton.ts

```diff
@@ -600,6 +600,16 @@ export class Skeleton extends UIRenderer {
     public onRestore () {
 
     }
+    /**
+     * @en Gets the animation state object.
+     * @zh 获取动画状态。
+     * @method getState
+     * @return {sp.spine.AnimationState} state
+     */
+    public getState(): spine.AnimationState | undefined {
```

Comment:
Why export this state?

Reply:
This is the interface already exposed by 3.7.3 and in use by users.
<img width="702" alt="Screenshot 2023-07-05 at 14 08 49" src="https://github.com/cocos/cocos-engine/assets/2398658/75bd9b2b-34ff-4e7f-aff7-e93df5649868">